contracts/interchain-token-service/tests/testdata/deploy_remote_canonical_token_succeeds_native_token.golden

```diff
@@ -0,0 +1,3 @@
+contract: Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVAX5)
```

Comment:
Please review the new golden files carefully to ensure they match the expectations of the tests